medusa/helpers/__init__.py

```diff
@@ -1574,11 +1533,13 @@ def get_disk_space_usage(disk_path=None, pretty=True):
 
 def get_tvdb_from_id(indexer_id, indexer):
 
-    session = make_session()
+    content_session = ContentSession()
+    session = MedusaSession()
```

Comment:
You have two sessions here, you should only have one.  All the content_session, text_session, etc need to be removed.  They reduce functionality with no gain.